src/index.js

```diff
@@ -1,14 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
```

Comment:
wanna install prettier extension for vs code so we don't get formatting issues every time (ex the single vs double quotes)

Reply:
I'll add a .prettierrc file so that it doesn't pull random global settings.